processor/src/operations/mod.rs

```diff
@@ -9,9 +9,9 @@ mod fri_ops;
 mod horner_ops;
 mod io_ops;
 mod stack_ops;
-mod sys_ops;
+pub(crate) mod sys_ops;
 mod u32_ops;
-mod utils;
+pub mod utils;
```

Comment:
Do `utils` need to `pub` rather than `pub(crate)`?

Reply:
No, fixed